app/js/controllers/admin/process.js

```diff
@@ -0,0 +1,89 @@
+const AdminProcessCtrl = function ($scope, $state, $mdDialog, $mdToast, AdminDataService, ClientDataService, $sce) {
+
+    $scope.processes = getProcesses();
+    $scope.selected = [];
+    //utlility methods
+    $scope.trustAsHtml = (template) => {
+        return $sce.trustAsHtml(template);
+    };
+
+    $scope.fetchProcesses = () => {
```

Comment:
I like the way you have done the same implementation for the projects part

`getProjects();`

```
   function getProjects(){
        ClientDataService.fetchProjects()
             .then((resp) => {
                 $scope.projects = resp.data.projects;
                 $scope.filteredProjects = resp.data.projects;
             }, (err) => {
                 console.log(err, 'ERROR');
             });
     }
```
its seems more cleaner and easier to understand